### PR TITLE
Planet dump refactor

### LINKED ISSUE #28: [planet-dump] assert planet dump works

### PR DESCRIPTION
**Pull request by CptSchnitz**
closes #28

### CODE REVIEW

File: images/planet-dump/start.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    log.info('creating dump')
+    start_time = time.perf_counter()
+
+    run_command(create_dump_command, process_log.info,
+                process_log.error, handle_osmosis_failure, lambda: None)
+
+    end_time = time.perf_counter()
+
+    run_time = end_time - start_time
+    log.info('dump {0} as been created succesfuly, it took {1:0.4f} seconds'.format(
```
Comment (syncush) [inline images/planet-dump/start.py]: successfully

File: helm/microcosm/values.yaml [hunk 2]
```diff
@@ -139,11 +139,19 @@ dbbackup:
...
+  persistenceDisk:
+    enabled: true
+    accessMode: ReadWriteMany
+    mountPath: /mnt/dump
+    subPath: planet-dumps
+    # In case cloudProvider: aws
+    storageSize: 5Gi
   env:
-    CLEAN_BACKUPS: true
+    CREATE_DUMP_SCHEDULE: '0 0 * * *'
```
Comment (syncush) [inline helm/microcosm/values.yaml]: add cron to name

File: images/planet-dump/start.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        pause.until(execute_time)
+        create_dump()
+
+
+if __name__ == '__main__':
+    os.makedirs('/var/log/osm-seed', exist_ok=True)
+    log = JSONLogger(
+        'main-debug', additional_fields={'service': 'planet-dumper'})
+    process_log = JSONLogger(
+        'main-debug', additional_fields={'service': 'planet-dumper', 'description': 'osmosis logs'})
```
Comment (syncush) [inline images/planet-dump/start.py]: Please override the default config with a better file path and name

**Review by syncush — CHANGES_REQUESTED** [review]

File: images/planet-dump/start.py (hunk 1 above)
Comment (CptSchnitz) [reply images/planet-dump/start.py]: done

File: helm/microcosm/values.yaml (hunk 2 above)
Comment (CptSchnitz) [reply helm/microcosm/values.yaml]: done

File: images/planet-dump/start.py (hunk 3 above)
Comment (CptSchnitz) [reply images/planet-dump/start.py]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**